code/modules/mob/living/carbon/superior_animal/superior_defines.dm

```diff
@@ -110,7 +110,7 @@
 	/// Alpha value of our telegraph beam.
 	var/telegraph_beam_alpha = 50
 
-	var/moved = FALSE
+	// var/moved = FALSE         This appears to be unused. Commenting out for now.
```

Comment:
This seems to have been made unused due to the change seen in superior_animal/attack.dm line 12, please consider modifying comment.